pkg/hintrunner/zero/zerohint_others_test.go

```diff
@@ -531,5 +531,53 @@ func TestZeroHintOthers(t *testing.T) {
 				}),
 			},
 		},
+		"NondetElementsOverTwo": {
+			{
+				operanders: []*hintOperander{
+					{Name: "n", Kind: apRelative, Value: feltUint64(1)},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newNondetElementsOverTWoHint(
+						ctx.operanders["n"],
+					)
+				},
+				check: apValueEquals(feltUint64(0)),
+			},
+			{
+				operanders: []*hintOperander{
+					{Name: "n", Kind: apRelative, Value: feltUint64(2)},
```

Comment:
It does not really matter but can we maybe add a test for greater than case also, just to make them exhaustive ?

Reply:
tbh i think its redondant with =

Reply:
Hm... I guess the current tests are fine.